pulp_ansible/app/models.py

```diff
@@ -365,6 +365,12 @@ class AnsibleRepository(Repository):
 
     last_synced_metadata_time = models.DateTimeField(null=True)
     gpgkey = models.TextField(null=True)
+    last_sync_task = models.ForeignKey(
+        Task,
+        on_delete=models.CASCADE,
+        blank=True,
+        null=True,
+    )
```

Comment:
```suggestion
    last_sync_task = models.ForeignKey(
        Task,
        on_delete=models.CASCADE,
        null=True,
    )
```
`null=True` should be enough for a ForeignKey